INSTRUCTION.md

```diff
@@ -0,0 +1,12 @@
+# ToDo App Docker Instructions
+
+## Docker Hub Repository
+- The Docker image is available at: https://hub.docker.com/r/evgeniychernyak/todoapp:1.0.0
+
+## Build and Run the Docker Image
+
```

Comment:
The instructions for building and running the Docker image are incomplete. Please add the steps to build the Docker image using `docker build -t todoapp .` and to run the Docker image using `docker run -p 8080:8080 todoapp`.